extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -157,14 +159,69 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
       : FALLBACK_DEFAULT_STICKER;
   }
 
+  /**
+   * @private
+   */
+  maybeInitializeStickerStyle_() {
+    this.toggleStickerState_(
+      'outline',
+      this.element.getAttribute('sticker-style') === 'outline'
+    );
+    this.toggleStickerState_(
+      'dropshadow',
+      this.element.getAttribute('sticker-style') === 'dropshadow'
+    );
+
+    const outlineColor = this.element.getAttribute('outline-color');
+    if (this.isValidRGBColor_(outlineColor)) {
+      setImportantStyles(this.element, {
+        '--story-audio-sticker-outline-color': outlineColor,
+      });
+    }
+    const dropshadowColor = this.element.getAttribute('dropshadow-color');
+    if (this.isValidRGBColor_(dropshadowColor)) {
+      setImportantStyles(this.element, {
+        '--story-audio-sticker-dropshadow-color': dropshadowColor,
+      });
+    }
+  }
+
+  /**
+   * @private
+   * @param {string} RGBColor
+   * @return {boolean}
+   */
+  isValidRGBColor_(RGBColor) {
```

Comment:
The variable should start with lowercase.

Also, remove the function from the class since it's stateless. Also, compile the regex as a constant at first, and use it here. Or, rely on AMP validation to do the regex so you don't have to validate yourself. Make sure to test thoroughly in any case.

Reply:
Just double checked and yes agree that to check this type of developer-provided attribute value, we should always rely on validator instead. Removed!

This JS check removed here should be for checking the finalized CSS value specified by the developer in their CSS that cannot be checked via validator, e.g. the one found in utils.js that also checks rgb color.